src/content.js

```diff
@@ -1,6 +1,8 @@
 Array.prototype.forEach.call (
-	document.querySelectorAll ('tr[data*=inherited]'),
+	document.querySelectorAll ('tr'),
 	function (node) {
-		node.style.display = node.style.display === 'none' ? '' : 'none';
+		if(node.innerHTML.indexOf('(Inherited from') != -1){
```

Comment:
Yes, they have removed their table row attributes. This is why I started searching for a solution and this is the one I found.
Feel free to modify it if you find other solution, but this code works and works fast. Additionally, I have added this workaround in the Chrome extension comments, so other users can, until this is fixed. ;)